src/components/Navigation.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import Link from 'next/link';
+import Image from 'next/image';
+import Logo from '../../public/logo.png';
+import styles from '../styles/Navigation.module.scss';
+
+const Navigation = () => {
+  return (
+    <StyledContainer className={styles.container}>
```

Comment:
kurš no veidiem tev pašam vislabāk patīk ?

Reply:
i would not combine styled components + modules on same component. Messy, you know